proxy/http/HttpTransact.cc

```diff
@@ -3460,7 +3460,7 @@ HttpTransact::HandleStatPage(State *s)
       s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, s->internal_msg_buffer_type, len);
     }
   } else {
-    s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, "text/plain", 9);
+    s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, "text/plain", 10);
```

Comment:
@jpeach I've created [TS-4451| https://issues.apache.org/jira/browse/TS-4451] for the longer-term fix for all of these magic numbers. For now I'm going to merge this in as this fixes it and matches "style" in its context.